elementary/monitor/data_monitoring.py

```diff
@@ -83,13 +84,32 @@ def __init__(
         self.send_test_message_on_success = send_test_message_on_success
         self.disable_samples = disable_samples
 
+    def _parse_emails_to_ids(self, owners_str: str):
+
+        def _regex_match_owner_email(potential_email_str):
+            email_regex = r'\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b'
+
+            return re.fullmatch(email_regex, potential_email_str)
+
+        if owners_str != [] and owners_str != '':
+            owners_list = [owner.strip() for owner in owners_str.split(',')]
+            owners_validated = [
+                f'<@{self.slack_client.get_user_id_from_email(owner)}>' if _regex_match_owner_email(owner) else owner
```

Comment:
You should check if the response from `get_user_id_from_email` is empty first, and in that case keep the owner as is (right now an invalid mail will cause you to save `<@>`)